eclipse/ts.eclipse.ide.core/src/ts/eclipse/ide/internal/core/preferences/TypeScriptCorePreferenceInitializer.java

```diff
@@ -222,4 +218,50 @@ public static void fixEmbeddedPreference(IEclipsePreferences preferences) {
 			}
 		}
 	}
+
```

Comment:
Please add EPL license like https://github.com/angelozerr/typescript.java/blob/master/core/ts.core/src/ts/TypeScriptKind.java and add your name.